tensorflow_lite_support/python/task/vision/image_classifier_test.py

```diff
@@ -0,0 +1,499 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for image_classifier."""
+
+from absl.testing import parameterized
+
+from tensorflow_lite_support.python.task.core import task_options
+from tensorflow_lite_support.python.task.processor import processor_options
+from tensorflow_lite_support.python.task.processor.proto import bounding_box_pb2
+from tensorflow_lite_support.python.task.vision import image_classifier
+from tensorflow_lite_support.python.task.vision.core import tensor_image
+from tensorflow_lite_support.python.test import test_util
+
+import unittest
+import textwrap
+
+
+_MODEL_FLOAT = "mobilenet_v2_1.0_224.tflite"
```

Comment:
@lu-wang-g @mbrenon @ziyeqinghan
I think it's fine to test with just one TFLite model file (e.g. only with the float model) as it's the responsibility of the underlying C++ layer to ensure that Task Library works correctly with both float and quantized model. WDYT?

Reply:
Agreed.